modules/tensor_mechanics/test/tests/czm/tests

```diff
@@ -0,0 +1,144 @@
+    'The gold file of this test is a symbolic link to czm_3DC_load_only_normal_out.e'
+    mesh_mode = 'REPLICATED'
+    recover = false
+    method = '!dbg' # issue 12033
+    design = 'syntax/Modules/TensorMechanics/CohesiveZoneMaster/index.md source/materials/czm/SalehaniIrani3DCTraction.md source/interfacekernels/CZMInterfaceKernel.md source/actions/CohesiveZoneModelAction.md'
+  [../]
+  [./czm_3DC_load_complex_Action]
+    type = Exodiff
+    input = czm_3DC_load_complex_Action.i
+    exodiff = czm_3DC_load_complex_Action_out.e
+    issues = '#11546'
+    requirement = 'The CZMAction must be able to setup the 3 required CZMInterfaceKernel instances'
+    'and replicate the results obtained by setting up the input file manually under complex loading conditions'
+    'The gold file of this test is a symbolic link to czm_3DC_load_complex_out.e'
+    mesh_mode = 'REPLICATED'
+    recover = false
+    method = '!dbg' # issue 12033
+    design = 'syntax/Modules/TensorMechanics/CohesiveZoneMaster/index.md source/materials/czm/SalehaniIrani3DCTraction.md source/interfacekernels/CZMInterfaceKernel.md source/actions/CohesiveZoneModelAction.md'
+  [../]
+  [./czm_3DC_load_Normal_2D]
+    type = Exodiff
+    input = czm_3DC_load_only_normal_2D.i
+    exodiff = czm_3DC_load_only_normal_2D_out.e
+    issues = '#11546'
+    requirement = 'The system shall allow for cohesive zone laws to'
+    'represent the traction-separation behavior at an interface between'
+    'two bodies represented by continuum elements in 2D. By construction of the test,'
+    'only a normal gap should appear. The maximum theoretical '
+    'normal traction of 100 MPa shall be seen when the normal gap reaches a value of 1'
+    mesh_mode = 'REPLICATED'
+    recover = false
+    method = '!dbg' # issue 12033
+    design = 'source/materials/czm/SalehaniIrani3DCTraction.md source/interfacekernels/CZMInterfaceKernel.md'
+  [../]
+  [./czm_3DC_load_Normal_2D_Action]
+    type = Exodiff
+    input = czm_3DC_load_only_normal_2D_Action.i
+    exodiff = czm_3DC_load_only_normal_2D_Action_out.e
+    issues = '#11546'
+    requirement = 'The CZMAction must be able to setup the 2 required CZMInterfaceKernel instances'
+    'and replicate the results obtained by setting up the input file manually. '
+    'The gold file of this test is a symbolic link to czm_3DC_load_only_normal_2D_out.e'
+    mesh_mode = 'REPLICATED'
+    recover = false
+    method = '!dbg' # issue 12033
+    design = 'syntax/Modules/TensorMechanics/CohesiveZoneMaster/index.md source/materials/czm/SalehaniIrani3DCTraction.md source/interfacekernels/CZMInterfaceKernel.md source/actions/CohesiveZoneModelAction.md'
+  [../]
+  [./czm_3DC_load_Normal_1D]
+    type = Exodiff
+    input = czm_3DC_load_only_normal_1D.i
+    exodiff = czm_3DC_load_only_normal_1D_out.e
+    issues = '#11546'
+    requirement = 'The system shall allow for cohesive zone laws to'
+    'represent the traction-separation behavior at an interface between'
```

Comment:
The system shall allow for cohesive zone laws to represent the traction-separation behavior at an interface between two bodies represented by continuum elements in 1D using the Salehani Irani 3DC model, and only compute a normal gap under purely normal loading.